k8s/charts/rtsp-camera-simulator/values.yaml

```diff
@@ -0,0 +1,7 @@
+image:
+  repository: "devops-repo.isus.emc.com:8116/nautilus/gstreamer:pravega-dev"
```

Comment:
This is an internal server and it should not be referenced in OSS code. Make the default "gstreamer:pravega-dev" and a comment that the user should change it.